rsa/parallel.py

```diff
@@ -84,7 +84,44 @@ def getprime(nbits, poolsize):
     return result
 
 
-__all__ = ['getprime']
+def exec_parallel_curry(function,poolsize):
```

Comment:
This seems like a self-implemented variant of `functools.partial()` -- use that instead.